enterprise/admin/views.py

```diff
@@ -229,28 +232,47 @@ def _enroll_users(cls, enterprise_customer, emails, course_id, mode, request):
             course_id: The ID of the course in which we want to enroll
             mode: The enrollment mode the users will be enrolled in the course with
             request: The HTTP request the enrollment is being created by
+            notify: Whether to notify (by email) the users that have been enrolled
         """
         enrolled = []
         non_existing = []
         failed = []
         enrollment_client = EnrollmentApiClient()
+        course_details = CourseApiClient().get_course_details(course_id)
+        reverse_method = get_reverse_by_site(request, enterprise_customer.site)
+        course_url = reverse_method('about_course', args=(course_id,))
```

Comment:
In re: your code suggestion. I'd rather not do it that way; it assumes that the site is accessible over https. My code here makes an assumption too; that the other site can be accessed using the same scheme as the admin site, but I think that's just enough safer that I'd rather keep it, even though it's more complicated.

> Actually, on second thought, this assumes that the enterprise app and the LMS are deployed into the same django project. Does the Course catalog API not provide the URL of the course?

As far as I can tell from available information, it does not. I've done extensive testing, and have been unable to locate anything that suggests it does.